packages/core/src/modules/dids/__tests__/peer-did.test.ts

```diff
@@ -102,15 +102,20 @@ describe('peer dids', () => {
     expect(peerDid.didDocument).toMatchObject(didPeer1zQmY)
 
     // Save the record to storage
-    const didDocumentRecord = new DidDocumentRecord({
+    const didDocumentRecord = new DidRecord({
       id: didPeer1zQmY.id,
       role: DidDocumentRole.Created,
       // It is important to take the did document from the PeerDid class
       // as it will have the id property
       didDocument: peerDid.didDocument,
+      tags: {
+        // We need to save the recipientKeys, so we can find the associated did
+        // of a key when we receive a message from another connection.
+        recipientKeys: peerDid.didDocument.recipientKeys,
+      },
```

Comment:
Main difference is that we need to store the recipientKeys so we can retrieve the connection associated with a received message